### ISSUE TITLE
[Feature Request] Option for hotkeys to work with all Retropads

### ISSUE THREAD

**Issue by n1ghttrap** [libretro/RetroArch#14740]
I would simply like for any player/joystick/gamepad I have plugged in to be able to fast forward, enter the menu, etc but this is only reserved for the keyboard and Retropad 1.   I feel like this should be fine without unforeseen issues with other players triggering hotkeys with the Hotkey Enable key assigned to a button not pressed often in play, such as guide or a stick click, combined with other buttons.  

Thanks for you time.

**Comment by Mangaclub** [comment]
Yes PLEASE! I have to make a work around a bug where retroarch randomly disables the F1 key and quiet key (other hotkeys work) here on my raspberry pi so i wanted to use a second controller (GPIO Controller) to act as Exit button till i found out that only player 1 can do so! WHY!?

**Comment by hizzlekizzle** [comment]
I think the complicating factor here is that the hotkeys are tied to individual keycodes rather than retropad buttons, so unless all of the pads have identical keycodes, the hotkeys would end up on unintended buttons (or no button at all).
